src/app-link/test/Link.test.js

```diff
@@ -0,0 +1,20 @@
+import { expect, fixture, html } from "@open-wc/testing";
+import { Link } from "../Link.js";
+
+const scopedElements = { "app-link": Link };
+const scopedFixture = (template) => fixture(template, { scopedElements });
+
+describe("Link", () => {
+  it("should be accessible", async () => {
+    const element = await scopedFixture(html`<app-link></app-link>`);
+    await expect(element).to.be.accessible();
+  });
+
+  it("should show a valid href when the user press the link", async () => {
+    const element = await scopedFixture(html`<app-link .href=${'/my-movies'}></app-link>`);
+
+    const link = element.shadowRoot.querySelector(".linkApp");
+
+    expect(link.href).to.be.equal('http://localhost:8000/my-movies');
```

Comment:
This this may fail if your web test runner launches in a different host or port, you should not test that kind of info in the tests of your component.

Checking that the href "includes" the desired route is more than enough:

expect(link.href.includes('/my-movies')).to.be.true;